pages/common/sponge.md

```diff
@@ -0,0 +1,12 @@
+# sponge
+
+> Soak up standard input and write to a file.
+> Sponge soaks up all its input before writing the output file.
+
+- Appended all file content to its origin file:
```

Comment:
```suggestion
- Append file content to the source file:
```
Something like this sounds better, but I stil don't understand what does `sponge` do here exactly?